hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -486,7 +487,9 @@ public void onExecutionFailure(Throwable e) {
      */
     public void logError(Throwable e) {
         final ILogger logger = getLogger();
-        if (e instanceof RetryableException) {
+        if (e instanceof SilentException) {
+            logger.finest(e.getMessage(), e);
+        } else if (e instanceof RetryableException) {
```

Comment:
Please ignore the remark about `RetryableException`, I now see it's just the logging that is being done here.